tutorials/vpn-interop-guides/interop-guide-template/index.md

```diff
@@ -0,0 +1,999 @@
+# Google Cloud VPN Interop Guide
+
+Using Cloud VPN with <vendor name><product name>Learn how to build site-to-site IPSEC VPN between Cloud VPN and `<vendor name><product name>`.
+
+<NOTE: Options or instructions are shown in angle brackets throughout this template. Change or remove these items as needed.>
+
+<Place vendor logo or equipment photo here>
+
+<Put trademark statements here>: <vendor terminology> and the <vendor> logo are trademarks of <vendor company name> or its affiliates in the United States and/or other countries.
+
+_Disclaimer: This interoperability guide is intended to be informational in nature and shows examples only. Customers should verify this information by testing it. _ 
+
+Author: <author name and email address>  
```

Comment:
Do you want this line to render as is, with author name and email address displayed?